scipy/stats/_new_distributions.py

```diff
@@ -331,6 +334,7 @@ def _mode_formula(self, *, a, b, ab, **kwargs):
         docs[dist_name] = _combine_docs(_module[dist_name])
     with open(_docpath, 'w') as f:
         json.dump(docs, f, indent="    ")
+    print("Successfully exported documentation")
```

Comment:
Added this because I wanted to test the documentation export system, and I figured that this will always be easier to get a message than checking the date of the file.